pkg/prometheus/testdata/AdditionalAlertRelabelConfigs.golden

```diff
@@ -1,3 +1,7 @@
 - action: drop
   source_labels: [__meta_kubernetes_node_name]
-  regex: spot-(.+)
\ No newline at end of file
+  regex: spot-(.+)
+- sourceLabels: [label1, label2]
+  separator: ""
+  action: replace
+  targetLabel: custom_label
```

Comment:
It took me forever to understand why such a simple change (changing separator to pointer), would change so much a golden file, e.g. source labels, action, etc.

That was until I realized that this file is not really a golden file by definition, but an input! We should change the suffix of this file to something else, e.g. `.input`

Reply:
:+1: btw the config format is wrong since it expects Prometheus snake-case format.